src/constant/network.ts

```diff
@@ -122,7 +122,7 @@ export const NetworkConfigs: {
     feeAddress: "08d8f59e475830d9a1bb97d74285c4d34c6dac08", // swth1prv0t8j8tqcdngdmjlt59pwy6dxxmtqgycy2h7
 
     eth: {
-      rpcURL: "https://ethereum-rpc.publicnode.com",
+      rpcURL: "",
```

Comment:
Now that I think about it, think just keep the `rpcURL` for the MetaMask class. Don't delete